### PR TITLE
Snapping and floater options in Advanced digitizing panel

### LINKED ISSUE #8239: [feature] Advanced Digitizing: more common angle options (Request in QGIS)
### Request for documentation
From pull request QGIS/qgis#52988
Author: @elpaso
QGIS version: 3.32

**[feature] Advanced Digitizing: more common angle options**

### PR Description:
- add 0.1, 0.5, 1.0 common angles
- add an option to show current common angle in the floater
- add 'N' 'SHIFT+N' keyboard shortcuts to cycle through the common angle options

cad-show-common-angle

cad-floater-show-common-angle

### Commits tagged with [need-docs] or [FEATURE]

### LINKED ISSUE #8265: Advanced digitizing: floater settings and bearing (Request in QGIS)
### Request for documentation
From pull request QGIS/qgis#53061
Author: @elpaso
QGIS version: 3.32

**Advanced digitizing: floater settings and bearing**

### PR Description:
1. add configuration settings for floater items display
2. add a new option for bearing/azimuth display in the floater

advanced-digitizing-bearing-floater-options

### Commits tagged with [need-docs] or [FEATURE]

### PR DESCRIPTION
**Pull request by DelazJ**
Fixes #8265, fixes #8239

### CODE REVIEW

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 1]
```diff
@@ -1730,14 +1730,33 @@ At the top of the :guilabel:`Digitizing panel`, you find the following buttons:
...
-  using the :ref:`panel's shortcuts <digitizing_panel_shortcuts>`, edited and
-  |locked| :sup:`Locked` after validation (pressing :kbd:`Enter`).
+
+  :ref:`Snapping to features <snapping_options>` can be used along with
+  snapping to common angles for accurate digitizing.
+  For a fine-grained control on how the target element to snap to is retained,
+  you can indicate whether to prioritize snapping to features over common angles,
+  and vice-versa under the :guilabel:`Snapping priority` entry.
+  You can switch from one method to the other during the digitizing operation,
+  and this avoids disabling any of the snapping options in the meantime.
```
Comment (DelazJ) [inline docs/user_manual/working_with_vector/editing_geometry_attributes.rst:1744]:
```suggestion
  and this avoids disabling any of the snapping options in the meantime.
  Press :kbd:`N` (or :kbd:`Shift+N`) during a digitizing operation to cycle through the history of your recent common angles selection.
```

Comment (DelazJ) [reply docs/user_manual/working_with_vector/editing_geometry_attributes.rst:1744]: @elpaso May I request your review for this PR and feedback on the above suggestions? Refs https://github.com/qgis/QGIS/issues/53713#issuecomment-1623487814. Thanks

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 2]
```diff
@@ -1730,14 +1730,33 @@ At the top of the :guilabel:`Digitizing panel`, you find the following buttons:
...
+  
+* |floater| :sup:`Floater settings`: if the :guilabel:`Show floater` item is checked,
+  a contextual menu with digitizing information follows the cursor during digitizing.
+  The values can be accessed using the :ref:`panel's shortcuts <digitizing_panel_shortcuts>`,
+  edited and |locked| :sup:`Locked` after validation (pressing :kbd:`Enter`).
+  The type of information to return can be selected in the bottom part of the menu:
```
Comment (elpaso) [inline docs/user_manual/working_with_vector/editing_geometry_attributes.rst:1750]:
```suggestion
  The type of information to display can be selected in the bottom part of the menu:
```

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst (hunk 1 above)
Comment (elpaso) [reply docs/user_manual/working_with_vector/editing_geometry_attributes.rst:1744]: @DelazJ just got back from holiday, I'm building on windows and I will test the N / SHIFT+N issue on windows shortly.

The rest o this PR LGTM, thanks!

**Pull request merged**